nannyml/drift/ranking.py

```diff
@@ -123,9 +162,30 @@ class AlertCountRanking(Ranking):
 
     ALERT_COLUMN_SUFFIX = '_alert'
 
+    def fit(
+        self,
+        drift_calculation_result: UnivariateResults,
+        performance_results: Union[
+            CBPEResults, DLEResults, PerformanceCalculationResults
+        ] = None,
+    ) -> pd.DataFrame:
+        """Fits the calculator so it can then ranks drifted features according to their impact.
```

Comment:
You can drop the docstring here, since it is an implementation of the abstract `fit` method it will take that docstring.